src/hooks/useRequest.js

```diff
@@ -23,15 +27,33 @@ export function useGet(reactQueryKey, urlKey, config) {
 }
 
 export function usePost(reactQueryKey, urlKey, dataParam, config) {
-  return useOperation(reactQueryKey, Constants.REQUEST.POST, urlKey, dataParam ? dataParam : null, config ? config : null);
+  return useOperation(
+    reactQueryKey,
+    Constants.REQUEST.POST,
+    urlKey,
+    dataParam ? dataParam : null,
+    config ? config : null,
+  );
```

Comment:
same thing here, is there a way we could improve this readability?

Reply:
You can look at the weconnect project to check the consequences of having this
(some random component which has a big nomeclature/interface will be all breaked down).

Reply:
This line is 124 characters long. I configured to have a maximum of 120. We can change to a greater number and it will be everything in one line.

Reply:
niceee
i think like, 200 it would be nice.
WYO?